src/app/search/search-notebook-table/search-notebook-table.component.html

```diff
@@ -0,0 +1,80 @@
+<div>
+  <div *ngIf="!dataSource">
+    <mat-progress-bar mode="indeterminate"></mat-progress-bar>
+  </div>
+  <mat-table
+    [hidden]="!dataSource"
+    [dataSource]="dataSource"
+    matSort
+    matSortActive="starredUsers"
+    matSortDirection="desc"
+    matSortDisableClear
+    data-cy="search-notebook-results-table"
+  >
+    <ng-container matColumnDef="name">
+      <mat-header-cell *matHeaderCellDef mat-sort-header>Name</mat-header-cell>
+      <mat-cell data-cy="notebookColumn" *matCellDef="let notebook">
+        <div>
+          <a [matTooltip]="notebook?.full_workflow_path" [routerLink]="'/notebooks/' + notebook?.full_workflow_path">{{
```

Comment:
There's a pipe for the router link

Reply:
There's merge conflict looming, between this and Kimberley's search UI mods on the release branch.  So, for now, I'd suggest we let this ride, as it may soon be adjusted/rewritten then.

Reply:
Thanks for the heads-up, will do